portal-api/src/v1/validation/fields/issue-facility/requested-cover-start-date.js

```diff
@@ -29,75 +22,79 @@ module.exports = (
     manualInclusionNoticeSubmissionDate: manualInclusionNoticeSubmissionDateTimestamp,
   } = deal.details;
 
-  const dealSubmissionDate = formattedTimestamp(dealSubmissionDateTimestamp);
-  const requestedCoverStartDate = formattedTimestamp(submittedValues.requestedCoverStartDate);
-  const manualInclusionNoticeSubmissionDate = formattedTimestamp(manualInclusionNoticeSubmissionDateTimestamp);
+  const dealSubmissionDate = new Date(Number.parseInt(dealSubmissionDateTimestamp, 10));
+  const requestedCoverStartDate = new Date(Number.parseInt(submittedValues.requestedCoverStartDate, 10));
+  const manualInclusionNoticeSubmissionDate = new Date(
+    Number.parseInt(manualInclusionNoticeSubmissionDateTimestamp, 10),
+  );
   // EC 15 is: 'Cover Start Date is no more than three months from the date of submission'
   const eligibilityCriteria15 = deal.eligibility.criteria.find((c) => c.id === 15);
   const canEnterDateGreaterThan3Months = eligibilityCriteria15 && eligibilityCriteria15.answer === false;
 
-  const formattedDealSubmissionDate = moment(dealSubmissionDate).format('Do MMMM YYYY');
-  const formattedManualInclusionNoticeSubmissionDate = moment(manualInclusionNoticeSubmissionDate).format('Do MMMM YYYY');
+  const formattedDealSubmissionDate = format(dealSubmissionDate, CONSTANTS.DATE_FORMATS.LONG_FORM_DATE);
+  const formattedManualInclusionNoticeSubmissionDate = format(
+    manualInclusionNoticeSubmissionDate,
+    CONSTANTS.DATE_FORMATS.LONG_FORM_DATE
+  );
 
-  const today = moment();
-  const formattedToday = moment(today).format('Do MMMM YYYY');
-  const todayPlus3Months = moment(today).add(3, 'month');
-  const todayPlus3MonthsFormatted = moment(todayPlus3Months).format('Do MMMM YYYY');
+  const today = new Date();
+  const formattedToday = format(today, CONSTANTS.DATE_FORMATS.LONG_FORM_DATE);
+  const todayPlus3Months = add(today, { months: 3 });
+  const todayPlus3MonthsFormatted = format(todayPlus3Months, CONSTANTS.DATE_FORMATS.LONG_FORM_DATE);
 
-  const dealSubmissionPlus3Months = moment(dealSubmissionDate).add(3, 'month');
-  const dealSubmissionPlus3MonthsFormatted = moment(dealSubmissionPlus3Months).format('Do MMMM YYYY');
-  const minSubmissionPlus3Months = moment(manualInclusionNoticeSubmissionDate).add(3, 'month');
-  const minSubmissionPlus3MonthsFormatted = moment(minSubmissionPlus3Months).format('Do MMMM YYYY');
+  const dealSubmissionPlus3Months = add(dealSubmissionDate, { months: 3 });
+  const minSubmissionPlus3Months = add(manualInclusionNoticeSubmissionDate, { months: 3 });
 
-  if (requestedCoverStartDate) {
+  if (submittedValues.requestedCoverStartDate) {
     if (dealSubmissionType === CONSTANTS.DEAL.SUBMISSION_TYPE.AIN) {
-      if (moment(requestedCoverStartDate).isBefore(dealSubmissionDate, 'day')) {
+      if (isBefore(requestedCoverStartDate, startOfDay(dealSubmissionDate))) {
```

Comment:
As per `supply-contract-conversion-rate-date-is-required-for-non-GBP.js` (what a mouthful!), if we're only caring about dates, lets work as `startOfDay` throughout please for simplicity -- what do you think?

Also same comments on this file as `supply-contract-conversion-rate-date-is-required-for-non-GBP.js`